.github/workflows/release.yml

```diff
@@ -105,6 +106,13 @@ jobs:
           mv ${{ env.RELEASE_ADDS }} ./dist
           7z a -ttar -so -an ./dist | 7z a -si ./${{ env.RELEASE_DIR }}/${{ env.RELEASE_BIN }}-${{ steps.get_version.outputs.VERSION }}-${{ env.MACOS_TARGET }}.tar.gz
 
+      - name: Create tarball (MacOS M1)
```

Comment:
Instead of M1, should we just say `Apple Silicon`? Since I presume we'll soon see, M2, etc.

Reply:
good call!